pyFV3/stencils/delnflux.py

```diff
@@ -16,7 +17,7 @@ def calc_damp(damp_c: Quantity, da_min: Float, nord: Quantity) -> Quantity:
             "current implementation requires damp_c and nord to have "
             "identical data shape and dims"
         )
-    data = (damp_c.data * da_min) ** (nord.data + 1)
+    data = np.power((damp_c.data * da_min), (nord.data + 1), dtype=Float)
```

Comment:
Should this get stencilized at some point?

Reply:
It could. It lives in the init so it probably why. But it's flimsy because we rely on the transparency of numpy & cupy to do CPU/GPU, something that will bite us sooner or later